src/chainparams.cpp

```diff
@@ -286,15 +286,15 @@ class SigNetParams : public CChainParams {
             vSeeds.emplace_back("178.128.221.177");
             vSeeds.emplace_back("v7ajjeirttkbnt32wpy3c6w3emwnfr3fkla7hpxcfokr3ysd3kqtzmqd.onion:38333");
 
-            consensus.nMinimumChainWork = uint256S("0x0000000000000000000000000000000000000000000000000000008546553c03");
-            consensus.defaultAssumeValid = uint256S("0x000000187d4440e5bff91488b700a140441e089a8aaea707414982460edbfe54"); // 47200
+            consensus.nMinimumChainWork = uint256S("0x000000000000000000000000000000000000000000000000000000de26b0e471");
+            consensus.defaultAssumeValid = uint256S("0x00000112852484b5fe3451572368f93cfd2723279af3464e478aee35115256ef"); // 78788
             m_assumed_blockchain_size = 1;
             m_assumed_chain_state_size = 0;
             chainTxData = ChainTxData{
-                // Data from RPC: getchaintxstats 4096 000000187d4440e5bff91488b700a140441e089a8aaea707414982460edbfe54
-                /* nTime    */ 1626696658,
-                /* nTxCount */ 387761,
-                /* dTxRate  */ 0.04035946932424404,
+                // Data from RPC: getchaintxstats 4096 0000003d9144c56ac110ae04a0c271a0acce2f14f426b39fdf0d938c96d2eb09
+                /* nTime    */ 1645631279,
+                /* nTxCount */ 1257429,
```

Comment:
This one is correct.

Reply:
I'm running ``reindex-chainstate` and `assumevalid=0` on mainnet/testnet/signet. So far signet has completed. It matches.

```
./src/bitcoin-cli -signet getchaintxstats 4096 0000003d9144c56ac110ae04a0c271a0acce2f14f426b39fdf0d938c96d2eb09
{
  "time": 1645631279,
  "txcount": 1257429,
  "window_final_block_hash": "0000003d9144c56ac110ae04a0c271a0acce2f14f426b39fdf0d938c96d2eb09",
  "window_final_block_height": 78789,
  "window_block_count": 4096,
  "window_tx_count": 338057,
  "window_interval": 2432697,
  "txrate": 0.1389638742514995
}
```

> `$ src/bitcoin-cli -signet getblockhash 78788`

Out of curiosity, why 78788 instead of 78789? (or 78787 per `The selected value must not be orphaned, so it may be useful to set the value two blocks back from the tip`)